inference-engine/src/low_precision_transformations/src/convert_subtract_constant.cpp

```diff
@@ -0,0 +1,98 @@
+//
+//   Constant
+//     | I8
+//   Convert     Constant
+//      \ FP16   / FP16
+//       Subtract    Constant
+//         \ FP16   / FP16
+//          Multiply
+//
+// Result:
+//
+//   Constant    Constant
+//     | I8      | I8
+//   Convert     Convert
+//      \ FP16   / FP16
+//       Subtract    Constant
+//         \ FP16   / FP16
+//          Multiply
+//
+ngraph::pass::low_precision::ConvertSubtractConstant::ConvertSubtractConstant(const std::vector<ngraph::element::Type>& constantPrecisions) {
+    auto weightsConstantWrapper = ngraph::pattern::wrap_type<opset1::Constant>(pattern::consumers_count(1));
+    auto weightsConvertWrapper = ngraph::pattern::wrap_type<opset1::Convert>({ weightsConstantWrapper }, pattern::consumers_count(1));
+    auto subtractConstantWrapper = ngraph::pattern::wrap_type<opset1::Constant>(pattern::consumers_count(1));
+    auto subtractWrapper = ngraph::pattern::wrap_type<opset1::Subtract>({ weightsConvertWrapper, subtractConstantWrapper }, pattern::consumers_count(1));
+    auto multiplyConstantWrapper = ngraph::pattern::wrap_type<opset1::Constant>(pattern::consumers_count(1));
+    auto multiplyWrapper = ngraph::pattern::wrap_type<opset1::Multiply>({ subtractWrapper, multiplyConstantWrapper }, pattern::consumers_count(1));
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher & m) -> bool {
+        const auto& opsMap = m.get_pattern_value_map();
+        const auto weightsConvert = opsMap.at(weightsConvertWrapper).get_node_shared_ptr();
+        const auto quantizePrecision = weightsConvert->input(0).get_element_type();
+        const auto dequantizationPrecision = weightsConvert->output(0).get_element_type();
+
+        // validation by Convert operation input precisions
+        if (!constantPrecisions.empty()) {
+            const ngraph::element::Type inputPrecision = quantizePrecision;
+            if (std::find(constantPrecisions.begin(), constantPrecisions.end(), inputPrecision) == constantPrecisions.end()) {
+                return false;
+            }
+        }
+
+        const auto subtractConstant = opsMap.at(subtractConstantWrapper).get_node_shared_ptr();
+        if (!NetworkHelper::isConvertable(subtractConstant, quantizePrecision)) {
+            return false;
+        }
+
+        auto resultSubtractConstant = NetworkHelper::round(subtractConstant, quantizePrecision);
+        if (NetworkHelper::isScalarLike(resultSubtractConstant)) {
+            resultSubtractConstant = NetworkHelper::toScalar(resultSubtractConstant);
+            if (op::util::constantIsEqualTo(resultSubtractConstant, 0.f)) {
+                resultSubtractConstant = nullptr;
+            }
+        }
+
+        if (resultSubtractConstant == nullptr) {
+            const auto multiply = opsMap.at(multiplyWrapper).get_node_shared_ptr();
+            const auto newMultiply = std::make_shared<opset1::Multiply>(weightsConvert, multiply->get_input_node_shared_ptr(1));
```

Comment:
Minor: do not traverse graph manually if possible. Please, use `opsMap.at(multiplyConstantWrapper).get_node_shared_ptr()` instead of `multiply->get_input_node_shared_ptr(1)`